bin/assigner/src/main.cpp

```diff
@@ -156,11 +159,62 @@ enum class print_column_kind {
     SELECTOR
 };
 
-template<typename ValueType, typename ContainerType>
-void fill_vector_value(std::vector<ValueType> &table_values, const ContainerType &table_col, typename std::vector<ValueType>::iterator start) {
-    std::copy(table_col.begin(), table_col.end(), start);
+template<typename Endianness>
+void print_size_t(
```

Comment:
why we can't use `crypto3::marshaling` write implementation: https://github.com/NilFoundation/marshalling/blob/c5a00412ba13be6097b921884b424897887dd286/include/nil/marshalling/types/integral/basic_type.hpp#L110

Reply:
I tried, seems like it doesn't work with ofstreams, only with arrays

Reply:
```
error: no type named 'iterator_category' in 'std::iterator_traits<std::basic_ostream<char>>'
[build]                             = std::is_same<typename std::iterator_traits<TIter>::iterator_category,
[build]                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
```

Reply:
expected, it doesn't works as is. Need to add implementation read/write for `ofstream::iterator`